File: 프로그래머스/unrated/181890. 왼쪽 오른쪽/왼쪽 오른쪽.java
import java.util.ArrayList;
import java.util.Arrays;
class Solution {
    public String[] solution(String[] str_list) {
         ArrayList<String> answerList = new ArrayList<>();

        for (int i = 0; i < str_list.length; i++) {
            if (str_list[i].equals("l")) {
                for (int j = 0; j < i; j++) {
                    answerList.add(str_list[j]);
                }
                break;
            } else if (str_list[i].equals("r")) {
                for (int j = i + 1; j < str_list.length; j++) {
                    answerList.add(str_list[j]);
                }
                break;
            }
        }

        String[] answer = new String[answerList.size()];
        answer = answerList.toArray(answer);

        return answer;
    }
}

File: 프로그래머스/0/181890. 왼쪽 오른쪽/왼쪽 오른쪽.js
function solution(str_list) {
    var answer = [];
    let lIdx = "";
    let rIdx = "";
    
    for(let i=0; i<str_list.length; i++){
        if(str_list[i] === "l" || str_list[i] === "r"){
            if(str_list[i] === "l"){
                answer = str_list.slice(0,i);
                break;
            }else{
                answer = str_list.slice(i+1);
                break;
            }
        }
    }
    return answer;
}